nextcord/components.py

```diff
@@ -330,18 +337,23 @@ class UserSelectMenu(SelectMenuBase):
         Defaults to 1 and must be between 1 and 25.
     disabled: :class:`bool`
         Whether the select is disabled or not. Defaults to ``False``.
+    defaults: Optional[List[Union[:class:`.Member`, :class:`nextcord.User`, :class:`.SelectDefault`]]]
```

Comment:
This seems to be named `defaults` but the attribute is `default_values`.

Reply:
resolved as I renamed every attribute to `.defaults` now, as you wanted consistency between them. It only gets added as a key now